client/src/components/App.jsx

```diff
@@ -55,9 +56,7 @@ class App extends React.Component {
       <div>
         <Search searchSubmit={this.searchSubmit} searchBarTextChange={this.searchBarTextChange}/>
         <Ratings />
-        {this.state.currentReviews.map((review, index) => {
-          return <Review key={index} review={review}/>
-        })};
+        <ReviewsList currentReviews={this.state.currentReviews}/>
```

Comment:
This refactor looks very clean. Nice work on keeping your App component less clustered